store/tikv/store_test.go

```diff
@@ -252,3 +254,53 @@ func (c *busyClient) SendCopReq(addr string, req *coprocessor.Request, timeout t
 	}
 	return c.client.SendCopReq(addr, req, timeout)
 }
+
+type mockPDClient struct {
+	sync.RWMutex
+	client pd.Client
+	stop   bool
+}
+
+func (c *mockPDClient) enable() {
+	c.Lock()
+	defer c.Unlock()
+	c.stop = false
+}
+
+func (c *mockPDClient) disable() {
+	c.Lock()
+	defer c.Unlock()
+	c.stop = true
+}
+
+func (c *mockPDClient) GetTS() (int64, int64, error) {
+	c.RLock()
+	defer c.RUnlock()
+
+	if c.stop {
+		return 0, 0, errors.New("stopped")
```

Comment:
Please define a new error ErrorStopped instead of creating a New error each time.